validator/versions.mk

```diff
@@ -15,5 +15,5 @@
 # Include global variables from gpu-operator project root
 include $(CURDIR)/../versions.mk
 
-CUDA_SAMPLES_VERSION ?= 11.7.1
+CUDA_SAMPLES_VERSION ?= 12.5.0
```

Comment:
Same question as before -- why was this change included in this PR? If it is not required, can we make this change in a separate PR?

Reply:
See [here](https://github.com/NVIDIA/gpu-operator/pull/937/files#r1722645761)